src/pipx/main.py

```diff
@@ -491,7 +493,7 @@ def _add_ensurepath(subparsers):
         action="store_true",
         help=(
             "Add text to your shell's config file even if it looks like your "
-            f"PATH already has {str(constants.LOCAL_BIN_DIR)}"
+            "PATH already contains pipx binary path(s)."
```

Comment:
I have been trying to avoid the word "binary" since it's kind of ambiguous. I try to use app or application instead. Is there something more precise that can be used here?

Reply:
Maybe “PATH alreay contains the 'pipx' command”?

Reply:
Maybe: `PATH already contains paths to pipx and pipx-installed apps.`

Does that sound good?

Reply:
(Just committed the above change.)